tests/user_messages_test.py

```diff
@@ -0,0 +1,74 @@
+from tic_tac_toe.user_messages import UserMessages
+from tests.printer_spy import PrinterSpy
+import pytest
+
+
+def test_user_interface_prints_logo_to_user():
+    spy = PrinterSpy()
+    user_message = UserMessages(spy)
+
+    user_message.logo()
+
+    expected = """
+    ________________   _________   ______   __________  ______
+   /_  __/  _/ ____/  /_  __/   | / ____/  /_  __/ __ \\/ ____/
+    / /  / // /        / / / /| |/ /        / / / / / / __/   
+   / / _/ // /___     / / / ___ / /___     / / / /_/ / /___   
+  /_/ /___/\\____/    /_/ /_/  |_\\____/    /_/  \\____/_____/"""
```

Comment:
🔥 Love it! This is the sort of constant I would expect to be defined once in the application, and imported for the application and for the test.